pkgs/tools/misc/todo/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, stdenv
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "todo";
+  version = "2.4";
+
+  src = fetchFromGitHub {
+    owner = "sioodmy";
+    repo = "todo";
+    rev = version;
+    sha256 = "Z3kaCNZyknNHkZUsHARYh3iWWR+v//JhuYoMIrq54Wo=";
+  };
+
+  cargoSha256 = "82xB+9kiLBwCE6yC3tlmgzJJgA1cMDq6Mjc48GBZ9B8=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl ];
+
```

Comment:
It looks like the binary gets installed as `todo-bin`, which is a bit inconsistent with other packages:
```suggestion
  preFixup = ''
    mv $out/bin/todo-bin $out/bin/todo
  '';